prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/tibco/TibjmsImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.parodos.tasks.tibco;
+
+import com.tibco.tibjms.TibjmsConnectionFactory;
+
+import javax.jms.*;
+
+public class TibjmsImpl implements Tibjms {
+
+	@Override
+	public void sendMessage(String url, String caFile, String username, String password, String topic, String message)
+			throws JMSException {
+		TibjmsConnectionFactory factory = new TibjmsConnectionFactory(url);
+		if (!caFile.isEmpty()) {
+			factory.setSSLTrustedCertificate(caFile);
+		}
+		Connection connection = factory.createConnection(username, password);
+		Session session = connection.createSession(javax.jms.Session.AUTO_ACKNOWLEDGE);
+		Destination destination = session.createTopic(topic);
+		MessageProducer producer = session.createProducer(destination);
+		TextMessage textMessage = session.createTextMessage(message);
+		producer.send(textMessage);
```

Comment:
The created resources (e.g. connection) are auto-closable. However, What makes them close?

Reply:
if it is auto-closable then what is the problem? I took this from TIBCO example/docs

Reply:
well, auto-closable works in try-with-resource which isn't the case here.
I'm afraid it may end up in leaking resources.